receiver/hostmetricsreceiver/example_config.yaml

```diff
@@ -0,0 +1,14 @@
+receivers:
+  hostmetrics:
+    scrape_interval: 10s
```

Comment:
Does the scrape interval applies to all of the scrapers? What is we want to have a `default` + per scraper config for this, so if the per scraper is not configured we use the "default".

Reply:
I hadn't really considered that use case tbh, but that does sound reasonable. I've updated the interfaces for now, and I'll make some changes to support this for the next PR